docs/data-sources/notification_destinations.md

```diff
@@ -0,0 +1,60 @@
+---
+subcategory: "Workspace"
+---
+# databricks_notification_destination Data Source
```

Comment:
we need to add `s` to the name: `databricks_notification_destinations`

Reply:
Done 👍